peripherals/fsmc/fsmc_sd.yaml

```diff
@@ -0,0 +1,85 @@
+# FMC as used in STM32F7
+
+"FMC,FSMC":
+  "SDCR?":
+    RPIPE:
+      NoDelay: [0, "No clock cycle delay"]
+      Clocks1: [1, "One clock cycle delay"]
+      Clocks2: [2, "Two clock cycles delay"]
+    RBURST:
+      Disabled: [0, "Single read requests are not managed as bursts"]
+      Enabled: [1, "Single read requests are always managed as bursts"]
+    SDCLK:
+      Disabled: [0, "SDCLK clock disabled"]
+      Mul2: [2, "SDCLK period = 2 x HCLK period"]
+      Mul3: [3, "SDCLK period = 3 x HCLK period"]
```

Comment:
I wonder if these two should be `Div2` and `Div3` since they decrease the clock frequency. I think so far we've only used `MulX` for _increasing_ clock frequency, and use `Div` even when the RM talks about period.